readme.md

```diff
@@ -0,0 +1,97 @@
+![](media/banner.png)
```

Comment:
These images need to be absolute URLs or else they won't work on https://www.npmjs.com/package/serve or locally

Reply:
Using absolute URLs now, but they won't work until the PR is merged into the `main` branch.